client/src/process/Zip.tsx

```diff
@@ -217,12 +217,12 @@ export class Zip {
   }
 
   async buildMediaMap() {
-    const {
-      tweet: tweets,
-      profile: { avatarMediaUrl: amu },
-    } = this.archiveItems;
-    let mediaId = amu.substring(amu.lastIndexOf("/") + 1, amu.length);
-    this.archiveItems.profile.avatarMediaUrl = this.media[mediaId];
+    const { tweet: tweets, profile } = this.archiveItems;
```

Comment:
What do you think of creating an interface for archiveItems and handle null / undefined amu while destructuring?